File: nuhiep/src/genre.lua

local doc = html:parse("<a href=\"/truyen?\">Tất cả</a><a href=\"/truyen?genre=10\">Hiện Đại Ngôn Tình</a><a href=\"/truyen?genre=13\">Huyền Huyễn Ngôn Tình</a><a href=\"/truyen?genre=14\">Tiên Hiệp Kỳ Duyên</a><a href=\"/truyen?genre=15\">Cổ Đại Ngôn Tình</a><a href=\"/truyen?genre=16\">Huyền Nghi Thần Quái</a><a href=\"/truyen?genre=17\">Khoa Huyễn Không Gian</a><a href=\"/truyen?genre=18\">Lãng Mạn Thanh Xuân</a>")
local genre = {}
if doc ~= nil then
    local el = doc:select("a")
    for i = 0, el:size() - 1 do
        local e = el:get(i)
        local link = {}
        link["title"] = e:text()
        link["input"] = "https://nuhiep.com" .. e:attr("href")
        link["script"] = "gen.lua"
        table.insert(genre, link)
    end
    return response:success(genre)
end

return nil

File: nuhiep/src/genre.js
function execute() {
    return Response.success([
        {title: "Tất cả", input: "https://nuhiep.com/truyen?", script: "gen.js"},
        {title: "Hiện Đại Ngôn Tình", input: "https://nuhiep.com/truyen?genre=10", script: "gen.js"},
        {title: "Huyền Huyễn Ngôn Tình", input: "https://nuhiep.com/truyen?genre=13", script: "gen.js"},
        {title: "Tiên Hiệp Kỳ Duyên", input: "https://nuhiep.com/truyen?genre=14", script: "gen.js"},
        {title: "Cổ Đại Ngôn Tình", input: "https://nuhiep.com/truyen?genre=15", script: "gen.js"},
        {title: "Huyền Nghi Thần Quái", input: "https://nuhiep.com/truyen?genre=16", script: "gen.js"},
        {title: "Khoa Huyễn Không Gian", input: "https://nuhiep.com/truyen?genre=17", script: "gen.js"},
        {title: "Lãng Mạn Thanh Xuân", input: "https://nuhiep.com/truyen?genre=18", script: "gen.js"}
    ]);
}